x-pack/plugins/enterprise_search/public/applications/enterprise_search/components/product_selector/product_selector.test.tsx

```diff
@@ -56,5 +59,12 @@ describe('ProductSelector', () => {
 
       expect(wrapper.find(ProductCard)).toHaveLength(0);
     });
+
+    it('renders the license and trial callouts', () => {
+      const wrapper = shallow(<ProductSelector access={{}} />);
+
+      expect(wrapper.find(TrialCallout)).toHaveLength(1);
+      expect(wrapper.find(LicenseCallout)).toHaveLength(1);
+    });
```

Comment:
This test got put under the `access checks` describe() block it looks like - can you move it outside of it / above line 34/33 for organization?

Reply:
Yep! Nice catch again. https://github.com/elastic/kibana/pull/99122/commits/61e6748718af06c3304087864247e57a22e31980